docs/source/examples/lora_finetune.rst

```diff
@@ -300,7 +302,80 @@ A comparison of the (smoothed) loss curves between this run and our baseline ove
     The above figure was generated with W&B. You can use torchtune's :class:`~torchtune.utils.metric_logging.WandBLogger`
     to generate similar loss curves, but you will need to install W&B and setup an account separately.
 
-As an exercise, you can also try running some evaluation tasks or manually inspecting generations
-output by your saved checkpoints (which can be found in :code:`output_dir`).
-You may want to train the model for longer first, as here we only looked at 500 steps
-(which corresponds to about 2% of one epoch of the Alpaca dataset).
+
+Trading off memory and model performance with LoRA
+--------------------------------------------------
+
+In the preceding example, we ran LoRA on two devices. But with LoRA's memory efficiency it is possible
+to run on a single device as well. This can be done via the command:
```

Comment:
```suggestion
In the preceding example, we ran LoRA on two devices. But given LoRA's low memory footprint, we can run fine-tuning on a single device using most commodity GPUs which support bfloat16 as a dtype. This can be done via the command:
```